presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -305,13 +305,22 @@ Property Name                                        Description
                                                      running in EC2, or when the catalog is in a different region.
                                                      Example: ``us-east-1``
 
+``hive.metastore.glue.endpoint-url``                 AWS Glue API endpoint URL.
+                                                     Example: ``https://glue.us-east-1.amazonaws.com``
+
 ``hive.metastore.glue.pin-client-to-current-region`` Pin Glue requests to the same region as the EC2 instance
                                                      where Presto is running, defaults to ``false``.
 
 ``hive.metastore.glue.max-connections``              Max number of concurrent connections to Glue,
                                                      defaults to ``5``.
 
-``hive.metastore.glue.default-warehouse-dir``        Hive Glue metastore default warehouse directory
+``hive.metastore.glue.default-warehouse-dir``        AWS Glue metastore default warehouse directory
```

Comment:
Not sure how to word this, but this is default warehouse directory used for new schemas, when using Glue, unless explicit location is specified.

@mosabua can you please suggest wording?

Reply:
default warehouse directory for newly created schemas with metadata storage in AWS Glue, can be overridden with the location property in the schema creation statement

Maybe something like thatt

Reply:
"Default warehouse directory for tables created without an explicit LOCATION property". I don't reiterating the AWS Glue part is necessary since it's all listed under the Glue heading.

Reply:
fair enough